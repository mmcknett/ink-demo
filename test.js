const React = require('react');
// const chalk = require('chalk');
const { render } = require('ink-testing-library');
const App = require('./ui');

it('greet unknown user', () => {
  const { lastFrame } = render(<App />);

  expect(lastFrame()).toMatchSnapshot();
});

it('greet user with a name', () => {
  const { lastFrame } = render(<App name="Jane" />);

  expect(lastFrame()).toMatchSnapshot();
});
