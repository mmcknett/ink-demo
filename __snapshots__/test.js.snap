// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`greet unknown user 1`] = `
"
                                          Time remaining:[32m10[39m


                  [34mMatt[39m's guesses                                    [34mAmy[39m's guesses

                  [7m [27m
"
`;

exports[`greet user with a name 1`] = `
"
                                          Time remaining:[32m10[39m


                  [34mMatt[39m's guesses                                    [34mAmy[39m's guesses

                  [7m [27m
"
`;
